module/VuFind/tests/unit-tests/src/VuFindTest/Log/Writer/Office365Test.php

```diff
@@ -49,17 +50,23 @@ class Office365Test extends \PHPUnit\Framework\TestCase
      *
      * @return void
      */
-    public function testWriter(): void
+    public function testHandler(): void
     {
-        // Set up data and expectations:
         $fakeUri = 'http://fake';
-        $expectedBody = '{"@context":"https:\/\/schema.org\/extensions",'
-            . '"@type":"MessageCard","themeColor":"0072C6",'
-            . '"title":"Test Title","text":"Formatted message."}';
-        $message = ['message' => 'test', 'priority' => 1];
+        $expectedBody = '{"@context":"https:\/\/schema.org\/extensions",' .
+            '"@type":"MessageCard","themeColor":"0072C6",' .
+            '"title":"Test Title","text":"[2025-07-09T14:55:20+00:00] test.INFO: test [] []\n"}';
```

Comment:
Our convention for multi-line statements is to put the operator on the start of the line rather than the end where possible:

```suggestion
        $expectedBody = '{"@context":"https:\/\/schema.org\/extensions",'
            . '"@type":"MessageCard","themeColor":"0072C6",'
            . '"title":"Test Title","text":"[2025-07-09T14:55:20+00:00] test.INFO: test [] []\n"}';
```